swagger.yaml

```diff
@@ -162,7 +162,7 @@ parameters:
       $ref: '#/definitions/UpdateVersion'
   limit:
     name: limit
-    description: "Maximum number of items that will be returned. A value of zero will return all available items, without limit. The default value is 100."
+    description: "Maximum number of items that will be returned. A value of zero will return zero items. The default value is 20."
```

Comment:
No mention of the default maximum limit? Im happy to explain this to you in more detail but see Maximum defaults bullet point in https://github.com/ONSdigital/dp/blob/master/standards/API_STANDARDS.md

Reply:
This will have to be implemented too, unless it is part of another story to extend the current functionality.